src/Select.js

```diff
@@ -122,7 +124,7 @@ const Select = React.createClass({
 			inputValue: '',
 			isFocused: false,
 			isLoading: false,
-			isOpen: false,
+			isOpen: this.props.isOpen !== null ? this.props.isOpen : false,
```

Comment:
I guess you're right, I was thinking in the case of `<Select isOpen={undefined}/>`, but I suppose React uses the default value in this case?